pkg/config/config.go

```diff
@@ -191,10 +192,6 @@ func ParseConfig(path string) (*Config, error) {
 		return nil, errors.Wrap(err, "validate config")
 	}
 
-	if len(cfg.ServerWhitelist) == 0 {
-		cfg.ServerWhitelist = append(cfg.ServerWhitelist, cfg.PublicAddress)
-	}
-
 	err = godotenv.Load(cfg.EnvFile)
```

Comment:
so after this can add that when ETH_PRIVATE_KEYS is not empty just parse it and add all pub keys to be whitelisted

Reply:
Done